### PR TITLE
[tests] Simplify assets tests at `SonataAdminExtensionTest`

### PR DESCRIPTION
**Pull request by phansys**
I am targeting this branch, because this guards BC.

## Subject

Use default assets configuration at `SonataAdminExtensionTest` in order to avoid hardcoded values.

### CODE REVIEW

**Comment by kunicmarko20** [comment]
This is tests, so changelog is not needed.

**Comment by phansys** [comment]
Should I remove the changelog then?

**Comment by kunicmarko20** [comment]
If you are only going to update tests, then yes.

**Comment by phansys** [comment]
Changelog removed. I see the "pedantic" label was added to this PR, can I know why? IMO, this is very disappointing.

**Comment by kunicmarko20** [comment]
https://github.com/sonata-project/SonataAdminBundle/blob/3.x/CONTRIBUTING.md#labelling-the-pr

**Comment by kunicmarko20** [comment]
Why is it disappointing? Do you think we maybe need to rename it? if you check the link I posted, you will understand why was that label added.

**Comment by phansys** [comment]
Thank you for the reference! Reading that, I think my disappoint depends on a personal perception regarding the [definition](https://www.google.com/search?q=define%3Apedantic), since the *excessively* word gives me the idea that the effort is not wanted:
>excessively concerned with minor details or rules; overscrupulous.

It's now clearer for me, but I don't have any proposal to rename the label now.
Thank you again for your clarification.

**Comment by kunicmarko20** [comment]
Thank you for your feedback.

File: tests/DependencyInjection/SonataAdminExtensionTest.php [hunk 1]
```diff
@@ -17,6 +17,25 @@
...
+     * @var string[]
+     */
+    private $defaultJavascripts = [];
+
+    protected function setUp()
+    {
+        parent::setUp();
+        $this->container->setParameter('kernel.bundles', []);
+        $this->load();
+        $this->defaultStylesheets = $this->container->getDefinition('sonata.admin.pool')->getArgument(3)['stylesheets'];
```
Comment (kunicmarko20) [inline tests/DependencyInjection/SonataAdminExtensionTest.php]: It seems that his line is over 120 characters, could you please break it? Thank you

File: tests/DependencyInjection/SonataAdminExtensionTest.php [hunk 2]
```diff
@@ -17,6 +17,25 @@
...
+        $this->defaultJavascripts = $this->container->getDefinition('sonata.admin.pool')->getArgument(3)['javascripts'];
```
Comment (kunicmarko20) [inline tests/DependencyInjection/SonataAdminExtensionTest.php]: It seems that his line is over 120 characters, could you please break it? Thank you

File: tests/DependencyInjection/SonataAdminExtensionTest.php [hunk 3]
```diff
@@ -65,208 +84,81 @@ public function testHasSecurityRoleParameters()
...
-            'foo/bar.css',
-            'bar/quux.css',
-        ]);
+        $this->assertSame(array_merge($this->defaultStylesheets, $extraStylesheets), $stylesheets);
     }
 
     public function testRemoveStylesheetsGetRemoved()
     {
         $this->container->setParameter('kernel.bundles', []);
+        $removeStylesheets = ['bundles/sonataadmin/vendor/admin-lte/dist/css/skins/skin-black.min.css', 'bundles/sonataadmin/vendor/jqueryui/themes/base/jquery-ui.css'];
```
Comment (kunicmarko20) [inline tests/DependencyInjection/SonataAdminExtensionTest.php]: It seems that his line is over 120 characters, could you please break it? Thank you

File: tests/DependencyInjection/SonataAdminExtensionTest.php [hunk 4]
```diff
@@ -65,208 +84,81 @@ public function testHasSecurityRoleParameters()
...
-            'foo/bar.js',
-            'bar/quux.js',
-        ]);
+        $this->assertSame(array_merge($this->defaultJavascripts, $extraJavascripts), $javascripts);
     }
 
     public function testRemoveJavascriptsGetRemoved()
     {
         $this->container->setParameter('kernel.bundles', []);
+        $removeJavascripts = ['bundles/sonataadmin/vendor/readmore-js/readmore.min.js', 'bundles/sonataadmin/jquery/jquery.confirmExit.js'];
```
Comment (kunicmarko20) [inline tests/DependencyInjection/SonataAdminExtensionTest.php]: It seems that his line is over 120 characters, could you please break it? Thank you

File: tests/DependencyInjection/SonataAdminExtensionTest.php [hunk 5]
```diff
@@ -65,208 +84,81 @@ public function testHasSecurityRoleParameters()
...
-        ]);
+        $this->assertSame(array_values(array_diff($this->defaultJavascripts, $removeJavascripts)), $javascripts);
     }
 
     public function testAssetsCanBeAddedAndRemoved()
     {
         $this->container->setParameter('kernel.bundles', []);
+        $extraStylesheets = ['foo/bar.css', 'bar/quux.css'];
+        $extraJavascripts = ['foo/bar.js', 'bar/quux.js'];
+        $removeStylesheets = ['bundles/sonataadmin/vendor/admin-lte/dist/css/skins/skin-black.min.css', 'bundles/sonataadmin/vendor/jqueryui/themes/base/jquery-ui.css'];
```
Comment (kunicmarko20) [inline tests/DependencyInjection/SonataAdminExtensionTest.php]: It seems that his line is over 120 characters, could you please break it? Thank you

File: tests/DependencyInjection/SonataAdminExtensionTest.php [hunk 6]
```diff
@@ -65,208 +84,81 @@ public function testHasSecurityRoleParameters()
...
+        $removeJavascripts = ['bundles/sonataadmin/vendor/readmore-js/readmore.min.js', 'bundles/sonataadmin/jquery/jquery.confirmExit.js'];
```
Comment (kunicmarko20) [inline tests/DependencyInjection/SonataAdminExtensionTest.php]: It seems that his line is over 120 characters, could you please break it? Thank you

File: tests/DependencyInjection/SonataAdminExtensionTest.php [hunk 7]
```diff
@@ -65,208 +84,81 @@ public function testHasSecurityRoleParameters()
...
-            'bundles/sonatacore/vendor/select2/select2.css',
-            'bundles/sonatacore/vendor/select2-bootstrap-css/select2-bootstrap.min.css',
-            'bundles/sonataadmin/vendor/x-editable/dist/bootstrap3-editable/css/bootstrap-editable.css',
-            'bundles/sonataadmin/css/styles.css',
-            'bundles/sonataadmin/css/layout.css',
-            'bundles/sonataadmin/css/tree.css',
-            'foo/bar.css',
-            'bar/quux.css',
-        ]);
+        $this->assertSame(array_merge(array_diff($this->defaultStylesheets, $removeStylesheets), $extraStylesheets), $stylesheets);
```
Comment (kunicmarko20) [inline tests/DependencyInjection/SonataAdminExtensionTest.php]: It seems that his line is over 120 characters, could you please break it? Thank you

File: tests/DependencyInjection/SonataAdminExtensionTest.php [hunk 8]
```diff
@@ -65,208 +84,81 @@ public function testHasSecurityRoleParameters()
...
-            'bundles/sonataadmin/vendor/masonry/dist/masonry.pkgd.min.js',
-            'bundles/sonataadmin/Admin.js',
-            'bundles/sonataadmin/treeview.js',
-            'bundles/sonataadmin/sidebar.js',
-            'foo/bar.js',
-            'bar/quux.js',
-        ]);
+        $javascripts = $this->container->getDefinition('sonata.admin.pool')->getArgument(3)['javascripts'];
+
+        $this->assertSame(array_merge(array_diff($this->defaultJavascripts, $removeJavascripts), $extraJavascripts), $javascripts);
```
Comment (kunicmarko20) [inline tests/DependencyInjection/SonataAdminExtensionTest.php]: It seems that his line is over 120 characters, could you please break it? Thank you

**Comment by greg0ire** [comment]
We are open to suggestions regarding the `pedantic` label :P maybe `awesome-but-not-worth-a-release`?

**Comment by kunicmarko20** [comment]
`you-shall-not-release`

**Comment by jordisala1991** [comment]
Thank you @phansys

**Pull request merged**